runtime/dart_snapshot.cc

```diff
@@ -57,7 +57,7 @@ static std::shared_ptr<const fml::Mapping> SearchMapping(
     const std::vector<std::string>& native_library_path,
     const char* native_library_symbol_name,
     bool is_executable) {
-#if FML_OS_IOS
+#if FML_OS_IOS || FML_OS_MACOSX
```

Comment:
Eventually we'll probably replace these with a SHOREBIRD_USES_ITERPRETER or something.